src/portfolio.js

```diff
@@ -211,11 +211,32 @@ const bigProjects = {
   projects: [
     {
       image: require('./assets/images/saayaHealthLogo.webp'),
-      link: 'http://saayahealth.com/',
+      projectName: "Saayahealth",
+      projectDesc: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+      footerLink: [
+        {
+          name: 'Visit Website',
+          url:
+            'http://saayahealth.com/',
+        },
+        {
+          name: 'Example',
```

Comment:
You can remove this example button right now, although leave a comment here mentioning that you can add extra buttons here.

Reply:
ok